CMakeLists.txt

```diff
@@ -39,6 +30,14 @@ endif()
 
 option(BUILD_SHARED_LIBS "Build a shared library" ON)
 
+# Get preCICE version from git (that assumes you are compiling on a git checkout)
```

Comment:
Either assert this assumption or handle the edge case.
Distributing archived source code without `.git` is a common example that does not satisfy this assumption.